core/tests/puppeteer-acceptance-tests/user-utilities/question-admin-utils.js

```diff
@@ -0,0 +1,308 @@
+  '.e2e-test-question-contributor';
+const viewRoleUserResult =
+  '.e2e-test-reviewer-roles-result';
+
+// "Add Contribution Rights" form elements.
+const addContributorUsernameInput =
+  'input#add-contribution-rights-user-input';
+const addContributonRightsCategorySelect =
+  'select#add-contribution-rights-category-select';
+const addContributionRightsSubmitButton =
+   'button#add-contribution-rights-submit-button';
+
+// "Remove Contribution Rights" form elements.
+const removeContributorUsernameInput =
+  'input#remove-contribution-rights-user-input';
+const removeContributonRightsCategorySelect =
+  'select#remove-contribution-rights-category-select';
+const removeContributionRightsSubmitButton =
+  'button#remove-contribution-rights-submit-button';
+
+module.exports = class QuestionAdmin extends baseUser {
+  /**
+   * Function for navigating to the contributor dashboard admin page.
+   */
+  async navigateToContributorDashboardAdminPage() {
+    await this.goto(contributorDashboardAdminUrl);
+  }
+
+  /**
+   * Function for adding a right of reviewing questions to a user.
+   * @param {string} username - the username of the user.
+   */
+  async addReviewQuestionRights(username) {
+    await this.type(addContributorUsernameInput, username);
+    await this.select(
+      addContributonRightsCategorySelect, reviewQuestionRightValue);
+    await this.clickOn(addContributionRightsSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function for adding a right of submitting questions to a user.
+   * @param {string} username - the username of the user.
+   */
+  async addSubmitQuestionRights(username) {
+    await this.type(addContributorUsernameInput, username);
+    await this.select(
+      addContributonRightsCategorySelect, submitQuestionRightValue);
+    await this.clickOn(addContributionRightsSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function for removng a right of reviewing questions to a user.
+   * @param {string} username - the username of the user.
+   */
+  async removeReviewQuestionRights(username) {
+    await this.type(removeContributorUsernameInput, username);
+    await this.select(
+      removeContributonRightsCategorySelect, reviewQuestionRightValue);
+    await this.clickOn(removeContributionRightsSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function for removing a right of reviewing questions to a user.
+   * @param {string} username - the username of the user.
+   */
+  async removeSubmitQuestionRights(username) {
+    await this.type(removeContributorUsernameInput, username);
+    await this.select(
+      removeContributonRightsCategorySelect, submitQuestionRightValue);
+    await this.clickOn(removeContributionRightsSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function to return the list of question reviewers
+   * @returns {object} displayedUsers - an element contains the list
```

Comment:
@imchristie I'm confused about this. What's the type of the return value? Is it a list of strings (rather than "an element containing the list" or some sort of object)?

If it's not a simple list of strings ... why not?

Reply:
The function $eval() returns an element which is wrapped into an ElementHandle so I put "object" but it is also an array with strings. It's a bit confusing so I will change the comments to string for a clearer description of what it returns. 
Also, I changed the comment of the return type of getContributionStatusForUser as it is a string of the result. And I changed the way of changing the return value to make it clearer.

Reply:
Hm, I still don't see any changes in the docstring?

Reply:
Done